src/fides/api/alembic/migrations/versions/4fba24045fec_add_property_id_indices.py

```diff
@@ -0,0 +1,66 @@
+"""add property_id indices
+
+Revision ID: 4fba24045fec
+Revises: 57479150efd6
+Create Date: 2024-05-17 19:25:36.539636
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from loguru import logger
+
+# revision identifiers, used by Alembic.
+revision = "4fba24045fec"
+down_revision = "57479150efd6"
+branch_labels = None
+depends_on = None
+
+def upgrade():
+    connection = op.get_bind()
+
+    # only run the index creation if the tables have less than 1 million rows
+    privacypreferencehistory_count = connection.execute(
+        sa.text("SELECT COUNT(*) FROM privacypreferencehistory")
+    ).scalar()
```

Comment:
Note this count can take a little while to fetch on larger tenants -

Reply:
I was looking into more efficient row count techniques but I wasn't sure if that was overkill, is there an approach that you recommend from your experience?

Reply:
there are ways to estimate a count https://stackoverflow.com/questions/7943233/fast-way-to-discover-the-row-count-of-a-table-in-postgresql, I also am not sure if this is overkill.